### ISSUE TITLE
Region free not working on game card

### ISSUE THREAD

**Issue by riverscn** [roxas75/rxTools#446]
RxTools 3.0 Beta 2 on a **EUR ** N3DS 10.3 Emunand.
Insert **US ** region game card. Icon is displayed in the slot, but simply get "this game card cannot be used" message while launching.

**Comment by hartmannaf** [comment]
There is another patch needed for gamecards, use the freeMultiPatcher or ntr cfw to use gamecards.

**Comment by stebrick** [comment]
I got the same problem on US console until I used freeMultiPatcher

But I have two further questions.

First, does this patch support online gameplay (under region free of coz) for retail cartridge?

Secondly, do I have to install NVER if my sysNAND is 9.2 and emuNAND is 10.7? I can get my JP pokemon (cartridge) working with my US console but it prompts me to update (inside the emuNAND of course) and this means I have to use Eshop on emuNAND 10.7.  I wonder it is advisable for me to update it in emuNAND (i haven't set up any NNID in the US console yet).

Any reply is much appreciated.

**Comment by urherenow** [comment]
You will just brick your emunand. Use NTR. If needed, learn about the language plugin.

Or... If you have a spare SD card, there are tutorials on this site for region swapping your emunand. You can have 1 card for US and another for Japan. This way, I'm pretty sure online play and such will actually work.

**Comment by stebrick** [comment]
Sorry, I am a bit confused.  How would I brick my emunand? You  mean by downloading update for the game on Eshop?

**Comment by urherenow** [comment]
I misunderstood. I was thinking system update. If it's a GAME update, it simply won't work (the update won't, but the game will work, and keep bugging you for an update). You may be able to use funkyCIA2 to grab the update though, and install it using BBM or Devmenu.

**Comment by stebrick** [comment]
Thanks for the clarification, urherenow.  I was very frightened when I heard about the "brick" and not dared to test anything.

OK, here is my result.  After using freeMultiPatcher on my New3DS XL (US), I could install the update of my JP cartridge game directly in emunand 10.7 (by rxtools) in my US Console.  No  It did not even prompt me to the eshop.  It also launch the game.   It works!!!  AND AWESOME~~~~
(haven't tested about the online gameplay so far)

However, I have to open up freeMultiPatcher manually every time after I boot up my system.  Could I get the patcher booted up automatically with my rxtools?

Much appreciated.

**Comment by urherenow** [comment]
No, sorry. If you had A9lh installed, you could install a couple of pre-patched files for permanent region free, but to my knowledge, it doesn't apply to carts; you'd still need FMP or NTR.

**Comment by stebrick** [comment]
Duly understood.   I installed NTR now.  Could it boot up automatically with rxtools?  If yes, how to do it?  It seems I do not have A9lh (not even know what it is at all).

**Comment by mariogamer2** [comment]
ARM11 réception handling are not supportés,so no direct boot...
